android/cli/hooks/aar-transform.js

```diff
@@ -41,9 +41,10 @@ const LIBRARY_ORIGIN_PORJECT = 'Project';
 
 exports.cliVersion = '>=3.2';
 
-exports.init = function (logger, config, cli) {
+exports.init = function (logger, config, cli, appc) {
```

Comment:
I sorted the arguments a little bit by their priority. `cli` and `builder` being the main objects i interact with, followed by utility stuff like `appc` and `logger` and the `callback` always being the last argument.